### ISSUE TITLE
two-fer: add reference to tutorial

### ISSUE THREAD

**Issue by FridaTveit** [exercism/java#1411]
We've decided that every exercise with difficulty less than 3 should reference the [tutorial](https://github.com/exercism/java/blob/master/exercises/hello-world/TUTORIAL.md) provided with the hello world exercise. This should make it easier for people to solve those early exercise. See issue #1389 for discussion. 

So please add a hints.md file in https://github.com/exercism/java/tree/master/exercises/two-fer/.meta with a link to the hello world tutorial and a brief explanation of what it is (e.g. "For more help on how to solve this exercise, please refer to the tutorial provided as part of the hello world exercise").

**Comment by jsertel** [comment]
Would like to take this one.

**Comment by PTrottier** [comment]
Has this issue been resolved?

**Comment by FridaTveit** [comment]
No, I don't think so. Feel free to work on it @PTrottier since there hasn't been any activity on it for a while 🙂

**Comment by PTrottier** [comment]
@FridaTveit When trying to execute the binary on Fedora 28 (Linux 64 bit), it just says "cannot execute binary file". I am planning to spin up a VM to finish the work.

**Comment by FridaTveit** [comment]
Which binary are you referring to @PTrottier? Configlet?

**Comment by PTrottier** [comment]
@FridaTveit Sorry, yes, for Configlet

**Comment by PTrottier** [comment]
I am planning to try using Ubuntu instead and see if there is any success

**Comment by sjwarner-bp** [comment]
@PTrottier a handful of different `configlet` releases can be found here:

https://github.com/exercism/configlet/releases

How did you download it? It is possible a broken link has taken you to the wrong release.

There is a script in the `java` repo under `scripts` to download the correct version of `configlet` for your machine.

**Comment by PTrottier** [comment]
@sjwarner-bp I used https://github.com/exercism/configlet/releases/download/v3.9.2/configlet-linux-64bit.tgz (the latest). 

> There is a script in the java repo under scripts to download the correct version of configlet for your machine.

Which script are you referring to:

- canonical_data_check.sh  
- increase-test-logging.sh  
- insert-ignores.sh  
- prune-extra-keep-files.sh

**Comment by sjwarner-bp** [comment]
@PTrottier Apologies, it lives in the `bin` directory. I would imagine it should work on your machine :slightly_smiling_face: 

That sounds like the correct version of configlet to be using though. I can't test it out right now, but if you continue to have these issues with configlet, it might be worth opening an issue and using configlet version 3.6.1.

I'm running Ubuntu 64 bit and use this version with no problem.

**Comment by FridaTveit** [comment]
Okay @PTrottier, let us know if there's anything we can do to help with your configlet issues! 🙂

**Comment by ymoskovits** [comment]
Hey everyone! I've never made an open source contribution before. Would I be able to try this one out? Is this a good place to start?

**Comment by FridaTveit** [comment]
That's great that you want to make an open source contribution @ymoskovits! 😄 Anything labelled `good first patch` should be good to start on, feel free to work on this one! And let us know if you have any issues at all! 🙂
